static/app/views/organizationGroupDetails/actions/index.tsx

```diff
@@ -417,12 +417,7 @@ class Actions extends Component<Props, State> {
           <ActionButton
             disabled={disabled}
             to={disabled ? '' : this.getDiscoverUrl()}
-            onClick={() => {
-              this.trackIssueAction('open_in_discover');
```

Comment:
can we add `this.trackIssueAction('open_in_discover');` to onRedirectDiscover

Reply:
o neat

Reply:
new PR